packages/common/src/events/api/types.ts

```diff
@@ -4,21 +4,18 @@ export {
   EventAttendanceType,
   EventStatus,
   GetEventsParams,
-  IAddress,
-  IAddressExtent,
-  IAddressLocation,
   IAssociation,
   ICreateAssociation,
   IOnlineMeeting,
   ICreateOnlineMeeting,
-  ICreateAddress,
   ICreateEvent,
-  ICreateEventGeometry,
   ICreateRegistration,
   IEvent,
   IEventPermission,
-  IEventCatalogItem,
-  IEventGeometry,
+  ILocation,
```

Comment:
Some of these interface/enum names are a bit ambiguous, while others are named more explicitly with `Event` in their name. We're not currently exporting anything from the event's `api` dir b/c the event `SortOrder` enum conflicts with one of the same name that's exported from the `discussions` subdir. Not sure the LOE, or if it's even possible given we're using an auto-generated client, to make some of these more explicit that they deal specifically with events? E.g. `IEventLocation`, `EventSortOrder`, etc?

Reply:
This can be adjusted